Базовая стилизация всех страниц

d7add9 opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/1954051/tasks/6)

:boom: https://htmlacademy-htmlcss.github.io/1954051-sedona-34/4/

d7add9 committed:
добавляет базовую стилизацию

Pakistanka commented on catalog.html:
```diff
@@ -4,13 +4,14 @@
     <nav class="navigation">
       <a class="navigation-logo" href="index.html">
-        <img class="page-header-logo" src="images/logo.svg" width="139" height="70" alt="Логотип Седоны.">
+        <img class="page-header-logo" src="images/logo.svg" width="139" height="70"" alt=" Логотип Седоны.">
```
`height="70""` образовалась вторая, ненужная кавычка
`alt=" Логотип Седоны."` - перед словом Логотип не нужен пробел

Pakistanka commented on styles/styles.css:
```diff
@@ -0,0 +1,234 @@
+  font-size: 30px;
+  line-height: 36px;
+  text-align: center;
+
```
Лишний enter

Pakistanka reviewed (changes requested):
Ты - молодец! Работа чистая и аккуратная)

Поправь две мелкие опечатки, и я смерджу

d7add9 committed:
исправляет опечатки и неточности

также начал сетки, на изменения, связанные с ними можно пока не обращать внимание )
